pyaedt/modeler/GeometryOperators.py

```diff
@@ -1448,3 +1448,79 @@ def orient_polygon(x, y, clockwise=True):
             x.reverse()
             y.reverse()
         return x, y
+
+    @staticmethod
+    @aedt_exception_handler
+    def degrees_over_rounded(numb, digits):
```

Comment:
@jeremich41 Can you rename ``numb`` as angle. Maybe ``number`` was your original thought.

Reply:
Yes, sure angle is better I did it without thinking about

Reply:
I accepted the other modifications but how I can get back these modifications on my branch to have it for next pull request after changing numb in angle?

Reply:
Hi @jeremich41 just open git bash in your local pyaedt repo and enter the command: `git pull`.
If you accept the modifications here on the web interface, changes are committed and push to origin.
Then you have to pull them (fetch origin and merge it with your local branch.)